dataclients/kubernetes/kube.go

```diff
@@ -470,7 +446,7 @@ func (c *Client) LoadUpdate() ([]*eskip.Route, []string, error) {
 		// TODO: use eskip.Eq()
 		if r, ok := next[id]; ok && r.String() != c.current[id].String() {
 			updatedRoutes = append(updatedRoutes, r)
-		} else if !ok && id != healthcheckRouteID && id != httpRedirectRouteID {
+		} else if !ok {
```

Comment:
Since healthcheck and globalredirect routes are all static we can move them into load `loadAndConvert()` and simplify update diff logic.

Reply:
Special handling of healthcheck route had caused problems in the past.
Previously healthcheck route was added to `updatedRoutes` after `loadAndConvert()` and until the #618 it happened on every poll cycle which caused constant routing table recreation https://github.com/zalando/skipper/issues/614#issuecomment-381357947

Static healthcheck and globalredirect routes means they do not change after initial `LoadAll()` and therefore will not end up neither in `updatedRoutes` nor in `deletedIDs`. They are also treated just like normal routes which simplifies the overall code flow.